Sources/NIONFS3/NFSReplyDecoder.swift

```diff
@@ -67,3 +67,6 @@ public struct NFS3ReplyDecoder: WriteObservingByteToMessageDecoder {
         self.procedures[data.rpcCall.xid] = data.rpcCall.programAndProcedure
     }
 }
+
+@available(*, unavailable)
+extension NFS3ReplyDecoder: Sendable {}
```

Comment:
Isn't it a bit inconsistent that we're marking some of these somewhat related types as Sendable and some as not Sendable? For example `NFS3ReplyDecoder` isn't, but `NFS3CallDecoder` is. I understand we may not have a need to make them all Sendable if we're not using them in a way that requires them to be Sendable, but given they're public API, users may expect some of these types to have matching sendability constraints.

Reply:
I don't think the consistency matters here: either the types are `Sendable` or they aren't. Making a type which _could_ be `Sendable` be not `Sendable` for consistency isn't a good reason IMO, it's just an unnecessary constraint.